hosts/common.nix

```diff
@@ -110,6 +110,15 @@ in
   # Shell
   programs.bash.completion.enable = true;
 
+  systemd.services.trim_profiles = {
+    description = "Delete older profiles";
+    serviceConfig.Type = "oneshot";
+    script = ''
+      /run/current-system/sw/bin/nix-env --delete-generations +5 --profile /nix/var/nix/profiles/system
```

Comment:
Suggested change (line 117):

```
      ${pkgs.nix}/bin/nix-env --delete-generations +3 --profile /nix/var/nix/profiles/system
```

That is:
- Use `${pkgs.nix}/bin/nix-env` instead of `/run/current-system/sw/bin/nix-env`
- Keep only the last 3 generations which to me seems enough for our use